composer.json

```diff
@@ -17,7 +17,7 @@
         "php": "^5.6 || ^7.0",
         "ext-json": "*",
         "ext-tokenizer": "*",
-        "composer/semver": "^1.4",
+        "composer/semver": "^1.4 || ^2.0",
```

Comment:
can we actually use only v2?

Reply:
I've checked for breaking changes and it would seem that everything will keep working alright: https://github.com/composer/semver/releases/tag/2.0.0

Reply:
@keradus that may bring conflicts with some other dependencies that are not supporting v2.

Reply:
`composer/semver` has no dependencies and I think none of our own dependencies depend on it, so it should be fine?

Reply:
I mean if some project has PHP CS Fixer as a dependency and another dependency which needs `composer/semver`, but only allows v1, then if we would drop v1 then after releasing this change that project could not update PHP CS Fixer.

Reply:
This is the same issue as with Symfony components: the solution is not to widen our dependencies version constraints but to discourage users from installing PHP CS Fixer via the `composer.json` of their projects. Assuming they don't, we can bump our dependencies the way we want.

Reply:
I agree that we should discourage users from installing PHP CS Fixer via composer. However if there is no need to restrict here why do it, there is no harm in it?